l10n_be_vat_reports/reports/be_vat_intra_consignment.xsd

```diff
@@ -0,0 +1,196 @@
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="ReplacedIntraListing" type="common:IntervatDeclarationReference_Type" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation xml:lang="fr">Identifiant/R�f�rence Intervat d'un relev� intracommunautaire pr�c�dent qui est remplac� par ce relev� (r�f�rence donn�e dans l'accus� de r�ception du relev� original)</xs:documentation>
+					<xs:documentation xml:lang="nl">Identificatie/Referentie Intervat van een vorige intracommunautaire opgave welke wordt vervangen door deze opgave(referentie terug te vinden in de ontvangstbevestiging van de originele opgave)</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Declarant" type="common:Declarant_Type">
+				<xs:annotation>
+					<xs:documentation xml:lang="fr">D�clarant (assujetti belge)</xs:documentation>
+					<xs:documentation xml:lang="nl">Aangever (Belgische belastingplichtige)</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Period" type="IntraPeriod_Type">
+				<xs:annotation>
+					<xs:documentation xml:lang="fr">P�riode couverte par le relev�</xs:documentation>
+					<xs:documentation xml:lang="nl">Periode waarop de opgave betrekking heeft</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="IntraClient" type="IntraClient_Type" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation xml:lang="fr">Informations concernant un client du relev� intracommunautaire</xs:documentation>
+					<xs:documentation xml:lang="nl">Informatie in verband met een klant in de intracommunautaire opgave</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="FileAttachment" type="common:FileAttachment_Type" minOccurs="0" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation xml:lang="fr">Informations concernant une annexe</xs:documentation>
+					<xs:documentation xml:lang="nl">Informatie betreffende een bijlage</xs:documentation>
+					<xs:documentation xml:lang="de">Informationen zu einer Anlage</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Comment" type="common:Comment_Type" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation xml:lang="fr">Commentaire libre</xs:documentation>
+					<xs:documentation xml:lang="nl">Vrije commentaar</xs:documentation>
+					<xs:documentation xml:lang="de">Freier Kommentar</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+            <xs:element name="Justification" type="common:Justification_Type" minOccurs="0" maxOccurs="unbounded">
+                <xs:annotation>
+                    <xs:documentation xml:lang="fr">Information contenant une justification</xs:documentation>
+                    <xs:documentation xml:lang="nl">a faire</xs:documentation>
+                    <xs:documentation xml:lang="de">a faire</xs:documentation>
+                </xs:annotation>
+            </xs:element>
```

Comment:
This part is not indented as the rest of the file.

But also there is missing translation. Is that OK ?